docs/Development/Patch OpenY.md

```diff
@@ -0,0 +1,38 @@
+Here you can find instructions how you can patch OpenY distribution used on 
+your project.
+
+# When you need to patch OpenY
+- In case you found a bug and prepared a patch for openY on github.
+- In case you developed a new feature that will be good to have in openY and 
+created Pull Request to openY repository
+- In case you want to add a feature that added to openY but not included yet to 
+openY release.
+
+# How to patch openY via composer?
+
+If you followed instructions [docs/Development/Start new OpenY project](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Start%20new%20OpenY%20project.md)
+and you have configured composer.json you need just to do a few simple steps:
```

Comment:
It's better to wrap composer.json into `` to indicate that it's file, lie this `composer.json`.